ui/component/or-map/src/mapwidget.ts

```diff
@@ -45,23 +48,29 @@ export class MapWidget {
     protected _loaded: boolean = false;
     protected _markersJs: Map<OrMapMarker, L.Marker> = new Map();
     protected _markersGl: Map<OrMapMarker, MarkerGL> = new Map();
+    protected _geoJsonSources: string[] = [];
+    protected _geoJsonLayers: Map<string, any> = new Map();
+    protected _realm?: string;
     protected _viewSettings?: ViewSettings;
     protected _center?: LngLatLike;
     protected _zoom?: number;
     protected _showGeoCodingControl: boolean = false;
     protected _showBoundaryBox: boolean = false;
     protected _useZoomControls: boolean = true;
+    protected _showGeoJson: boolean = true;
     protected _controls?: (Control | IControl | [Control | IControl, ControlPosition?])[];
     protected _clickHandlers: Map<OrMapMarker, (ev: MouseEvent) => void> = new Map();
     protected _geocoder?: any;
 
-    constructor(type: MapType, showGeoCodingControl: boolean, styleParent: Node, mapContainer: HTMLElement, showBoundaryBox = false, useZoomControls = true) {
+    constructor(type: MapType, styleParent: Node, mapContainer: HTMLElement, realm?: string, showGeoCodingControl: boolean = false, showBoundaryBox = false, useZoomControls = true, showGeoJson = true) {
```

Comment:
Realm should not be passed in (ideally components should not be realm aware and this should be handled at the higher app level); using `manager.displayRealm` is tolerable for now until some refactoring is done for view settings.

Why isn't `manager.displayRealm` sufficient?

Reply:
Hi! Just changed it so `or-map` is not realm aware anymore.

The initial intent was to fix geoJSON overlay and other glitches within the Appearance page of the Manager UI.
On that page you can edit mapsettings of each individual realm, making `manager.displayRealm` insufficient.
It caused `smartcity` geoJSON to be shown on the mapsettings of `manufacturer`, which is confusing.

I now changed it to a GeoJSON override property on `or-map`, similar to how we handle properties like `center`.
By default it will use the mapsettings of `manager.displayRealm`, with the Appearance page being the only exception.